bundler/helpers/v2/monkey_patches/definition_ruby_version_patch.rb

```diff
@@ -26,7 +26,7 @@ def source_requirements
         Gem::Specification.new("Ruby\0", requested_version)
     end
 
-    %w(2.5.3 2.6.10 2.7.8 3.0.7 3.1.5 3.2.4).each do |version|
+    %w(2.5.3 2.6.10 2.7.8 3.0.7 3.1.5 3.2.4 3.3.2).each do |version|
```

Comment:
The running Ruby version is already added by Bundler itself (`super` below), so it's not necessary to add 3.3.2 here, one version per minor Ruby is enough I believe.